README.rst

```diff
@@ -317,6 +317,20 @@ Load kernel modules and add them to `/etc/modules`:
             - tp_smapi
             - 8021q
 
+Configure kernel modules with additional options to `/etc/modprobe.d`
+following example will add `/etc/modprobe.d/nf_conntrack.conf` file
+with line `options nf_conntrack hashsize=262144`:
+
+.. code-block:: yaml
+
+    linux:
+      system:
+        kernel:
+          kmod_config:
```

Comment:
On my system, there's only 4 options:
```
$ cat /etc/modprobe.d/*|grep -Ev '^#'|awk '{print $1}'|sort -u
blacklist
install
options
softdep
```

Approach is to be specific, design exact metadata model - to avoid users doing anything and possibly abusing features by using in some different way that was intended.
Moreover final model is more readable and easier to modify or extend when using approach I described above.